requirements/dev.in

```diff
@@ -20,7 +20,7 @@ pytest-django
 black
 
 # To test async code
-pytest-asyncio
+pytest-asyncio==0.10.0  # required to pass test_new_job_transformation
```

Comment:
@sarah-clements FYI: you may be able to explain why this is needed

Reply:
I don't know. Maybe @armenzg does?